packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageView.kt

```diff
@@ -49,10 +49,27 @@ class ExpoImageView(context: ReactContext, private val requestManager: RequestMa
         }
     }
   }
+  private var propsChanged = false
   private var loadedSource: GlideUrl? = null
   internal var sourceMap: ReadableMap? = null
-  var blurRadius: Int? = null
-  var fadeDuration: Int? = null
+  internal var blurRadius: Int? = null
+    set(value) {
+      field = if (value != null && value > 0) {
+        value
+      } else {
+        null
+      }
+      propsChanged = true
+    }
+  internal var fadeDuration: Int? = null
+    set(value) {
+      field = if (value != null && value > 0) {
+        value
+      } else {
+        null
+      }
```

Comment:
```suggestion
      field = if (isValueValid(value)) value else null
```

Reply:
The same as above

```suggestion
      field = value.takeIf { it != null && it > 0 }
```
or

```suggestion
      field = value?.takeIf { it > 0 }
```